### PR TITLE
Docusaurify documentation

### PR DESCRIPTION
**Pull request by abailly-iohk**
This PR uses docusaurus to structure the documentation site. This PR is still in draft because I need to add github actions to publish it with the new style.

Screenshot 2023-02-07 at 13 19 30
Screenshot 2023-02-07 at 13 19 43

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,4 +22,39 @@ document](CONTRIBUTING.md).
 Issues can be filled in our [GitHub issue
 tracker](https://github.com/input-output-hk/ouroboros-consensus/issues).
 
+## Installation
```
Comment (dnadales) [inline README.md:25]: Perhaps this should be a sub-section of `Documentation`: once we add the Consensus code this could lead to confusion.

File: README.md [hunk 2]
```diff
@@ -22,4 +22,39 @@ document](CONTRIBUTING.md).
...
 
+
+This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
+
+### Local Development
+
+```
+$ yarn start
```
Comment (dnadales) [inline README.md:33]: I wonder if we should add a `.nix` file that provides a yarn installation (I could do it in a separate PR). I found it quite handy!

**Review by dnadales — APPROVED** [review]
Thank you!

File: README.md (hunk 2 above)
Comment (abailly-iohk) [reply README.md:33]: Definitely a good idea, and something easy to do (even I could do it :) )!

File: README.md (hunk 1 above)
Comment (abailly-iohk) [reply README.md:25]: We shall rewrite the `README` when we merge consensus code in this repo, I guess/
